src/main/java/com/happeningnow/Model/Comments.java

```diff
@@ -0,0 +1,73 @@
+package com.happeningnow.Model;
+
+import com.sun.jdi.PrimitiveValue;
+import jakarta.persistence.*;
+import org.hibernate.annotations.UuidGenerator;
+
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Table(name = "comments")
+public class Comments {
```

Comment:
Names for classes and tables should always be in singular.

Reply:
ok